opencsp/app/sofast/lib/sofast_common_functions.py

```diff
@@ -0,0 +1,232 @@
+"""
```

Comment:
This library is great, thanks for thinking of it! ❤️ I have a couple ideas of other things that might best belong here, see other comments. 
Only one picking thing: since it mostly relates to image acquisition and projection, what do you think about renaming this to something like: `sofast_comon_io_functions` or `sofast_common_system_functions` etc. for better clarity? Probably not necessary, and totally up to you!